src/Uno.UI/Controls/CommandBarRenderer.iOS.cs

```diff
@@ -117,11 +117,6 @@ protected override void Render()
 				: null;
 			Native.BackIndicatorImage = backButtonIcon;
 			Native.BackIndicatorTransitionMaskImage = backButtonIcon;
-
-			if (Element.Presenter != null)
```

Comment:
I want a 500-word essay on why this was originally added and why you've removed it :D

Reply:
It was something added in PR #431 (and never reviewed by the core Uno team) to fix a rendering problem after first load. This problem is not there anymore and was causing a bug because the Native.Frame is often zero there.

Reply:
It was actually added here https://github.com/unoplatform/uno/pull/357, seemingly to fix a similar bug to this one